index.html

```diff
@@ -241,5 +256,50 @@ <h2>Skills</h2>
         </ul>
       </div>
     </section>
+    <section class="form">
+      <p class="form-text">
+        <!-- I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. -->
+      </p>
+      <!-- This is the Form section of the Page with validations -->
+      <form class="form-inputs" action="https://formspree.io/f/xyyoqbgr" method="post">
+        <input type="text" class="form-name" name="name" placeholder="Full name" maxlength="30" required>
+        <input type="email" class="form-email" name="email" placeholder="Email address" required>
+        <textarea name="Message" class="form-comments" placeholder="Enter text here" cols="30" rows="10" maxlength="500" required></textarea>
+        <button type="submit" class="form-button">Get in touch</button>
+      </form>
```

Comment:
- [ ] Kindly make sure your buttons match the styling like on the Figma template like removing the `border`,... I suggest you apply it to all the buttons.

- [ ] Kindly make sure the Button interactions such as `hover`, `active` (when it's pressed), `disabled` and enable (default state) are styled as shown in the Figma template to stick to your chosen template. You can check all the details from the image below in Figma:
![btn_colors](https://user-images.githubusercontent.com/83042742/178602989-e796b568-a97c-4c8d-aaed-f3e08973993a.png)